src/main/java/com/minecraftabnormals/endergetic/core/events/CompatEvents.java

```diff
@@ -22,7 +22,7 @@
 	public static void onPotionExpire(PotionEvent.PotionExpiryEvent event) {
 		LivingEntity affected = event.getEntityLiving();
 		boolean isBabyEffect = event.getPotionEffect().getPotion() == ForgeRegistries.POTIONS.getValue(new ResourceLocation("savageandravage:shrinking"));
-		if (isBabyEffect || event.getPotionEffect().getPotion() == ForgeRegistries.POTIONS.getValue(new ResourceLocation("savageandravage:growth"))) {
+		if (isBabyEffect || event.getPotionEffect().getPotion() == ForgeRegistries.POTIONS.getValue(new ResourceLocation("savageandravage:growing"))) {
```

Comment:
`event.getPotionEffect().getPotion()` is called twice in this method so it'd be cleaner to have it be a local variable.

Reply:
Yes, probably. I'll change it now.